### DISCUSSION TITLE
Installation: wrong link in the instructions

### DISCUSSION THREAD

**Discussion by ilcane87** [Klace/stable-diffusion-webui-images-browser#2] [General]
Hi, first of all thanks a lot @Klace for already implementing the .txt parsing feature I asked for yesterday, it works like a charm!

I just wanted to notify people that the README.md file with the installation instructions for this fork actually has the link to the original project, it should be changed to this:
`git clone https://github.com/Klace/stable-diffusion-webui-images-browser`

On a side note, if this fork is going to be maintained, unlike the original repo which has gone silent for months, it may be a good idea to get it added to the official extensions list so people can actually find it, I just stumbled upon it by complete coincidence.

**Comment by Klace** [comment]
Corrected, thanks!!
